com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,76 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
```

Comment:
it's either x11 alone or wayland + fallback-x11. Not both.

Reply:
Never worked for me with fallback-x11 option. Will remove x11 option.

Reply:
Fixed